File: OCH_Tao/ABC/A/146.js
const S = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0];
const L = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
const X = L.indexOf(S);
console.log(7 - X);

File: OCH_Tao/ABC/A/146.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  string S;
  cin >> S;
  vector<string> X = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
  cout << 7 - distance(X.begin(), find(X.begin(), X.end(), S)) << endl;
}